modulemd/meson.build

```diff
@@ -529,3 +529,16 @@ if with_docs
         install : true,
     )
 endif
+
+help2man_opts = [
+  '--no-info',
+  '--section=1',
+]
+custom_target(
```

Comment:
I think this target needs to be under the with_manpages condition, as it can't be generated when help2man is not found.

Reply:
@jon-turney

from [`meson.build`](https://github.com/fedora-modularity/libmodulemd/blob/master/meson.build#L86)
```
with_manpages = get_option('with_manpages')
help2man = find_program('help2man', required: with_manpages)
```

This means that if `with_manpages` is set to `enabled`, the `meson configure` step will fail with an error (since the program is required) if `help2man` cannot be located. If `with_manpages` is `auto` (the default), it means that it will be built if `help2man` happens to be available, but if it's not, the `help2man` variable will be a [`disabler()`](https://mesonbuild.com/Disabler.html#page-description) object. A `disabler()` object means that any target that requires it will simply be skipped. If `with_manpages` is `disabled`, a `disabler()` object is always returned.

Reply:
Thanks.

But I'm not sure if that's quite right.   Don't you need to write `find_program('help2man', required: with_manpages, disabler: true)` to get a dsabler object?

I any event, this doesn't appear to be currently working as you expect: see https://travis-ci.org/jon-turney/meson-corpus-test/jobs/648623888.

Is this a meson regression (i.e. you've seen this work as expected for some meson version?).  Or a problem with the meson documentation?